web/api/v1/api.go

```diff
@@ -255,7 +255,17 @@ func respond(w http.ResponseWriter, data interface{}) {
 
 func respondError(w http.ResponseWriter, apiErr *apiError, data interface{}) {
 	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(422)
```

Comment:
Oddly enough I explained the choice of `422` in the design doc and of the ~10 people commenting no one seemed to disagree. That's why we have design docs, so we don't mess with APIs after v1.0.